### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by LocutusOfBorg**
two typo fixes

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
Thanks!

Not sure about the "one" fix, though. Any native english speaker who can comment?

**Comment by LocutusOfBorg** [comment]
https://lintian.debian.org/tags/spelling-error-in-manpage.html

**Comment by LocutusOfBorg** [comment]
I don't know too :p

File: docs/internals/data-structures.rst [hunk 1]
```diff
@@ -374,7 +374,7 @@ needs to be handled.
 
 Lastly, cache_ invalidation is handled by noting which feature
 flags were and which were not understood while manipulating a cache.
-This allows to detect whether the cache needs to be invalidated,
+This allows one to detect whether the cache needs to be invalidated,
```
Comment (jdchristensen) [inline docs/internals/data-structures.rst]: "allows borg to"?

File: docs/internals/data-structures.rst [hunk 2]
```diff
@@ -1022,7 +1022,7 @@ Before feeding the checksum algorithm any data, the file name (i.e. without any
 is mixed into the checksum, since the name encodes the context of the data for Borg.
 
 The various indices used by Borg have separate header and main data parts.
-IntegrityCheckedFile allows to checksum them independently, which avoids
+IntegrityCheckedFile allows one to checksum them independently, which avoids
```
Comment (jdchristensen) [inline docs/internals/data-structures.rst]: "allows borg to"

File: docs/man/borg-key-import.1 [hunk 3]
```diff
@@ -35,7 +35,7 @@ level margin: \\n[rst2man-indent\\n[rst2man-indent-level]]
 borg [common options] key import [options] [REPOSITORY] [PATH]
 .SH DESCRIPTION
 .sp
-This command allows to restore a key previously backed up with the
+This command allows one to restore a key previously backed up with the
```
Comment (jdchristensen) [inline docs/man/borg-key-import.1]: "This command restores a key ..."?

File: docs/usage/key_import.rst.inc [hunk 4]
```diff
@@ -54,7 +54,7 @@ borg key import
 Description
 ~~~~~~~~~~~
 
-This command allows to restore a key previously backed up with the
+This command allows one to restore a key previously backed up with the
```
Comment (jdchristensen) [inline docs/usage/key_import.rst.inc]: "This command restores a key ..."?

File: src/borg/archiver.py [hunk 5]
```diff
@@ -2650,7 +2650,7 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
                                help='Create an html file suitable for printing and later type-in or qr scan')
 
         key_import_epilog = process_epilog("""
-        This command allows to restore a key previously backed up with the
+        This command allows one to restore a key previously backed up with the
```
Comment (jdchristensen) [inline src/borg/archiver.py]: Same here.

**Comment by jdchristensen** [comment]
"allows to" definitely needs to be "allows one to" or "allows the user to" or something like that.  In some cases, "allows borg to" seems to make more sense.  And in other cases, maybe the sentence should be rephrased.  I just submitted some comments on the pull request.

**Comment by LocutusOfBorg** [comment]
pushed, let me know if it does fix your concerns!

**Comment by jdchristensen** [comment]
LGTM.

**Pull request merged**